frontend/src/sentry.js

```diff
@@ -1,8 +1,10 @@
 import * as Sentry from '@sentry/browser';
 
-var dsn = import.meta.env.VITE_SENTRY_FRONT_DSN;
+const dsn = import.meta.env.VITE_SENTRY_FRONT_DSN;
+export const isSentryActivated = import.meta.env.PROD && Sentry && dsn
```

Comment:
I fear there might be a possible confusion between the two `isSentryActivated` maybe we could have distinct names like `isSentryReactActivated`vs `isSentryViteActivated`